src/inputs/classes.scss

```diff
@@ -36,6 +36,14 @@
   }
 }
 
+.iui-input-label {
```

Comment:
Hm, should it have `input` in the name? We/Users might want to use label in other places just to have the same styling.

Reply:
`iui-component-label`? 😁